test/e2e/scheduling/predicates.go

```diff
@@ -135,7 +132,7 @@ var _ = SIGDescribe("SchedulerPredicates [Serial]", func() {
 				nodeMaxAllocatable = allocatable.Value()
 			}
 		}
-		WaitForStableCluster(cs, masterNodes)
+		WaitForStableCluster(cs, workerNodes)
```

Comment:
How about updating the parameter `masterNodes` of WaitForStableCluster() and getScheduledAndUnscheduledPods to `workerNodes`?

Reply:
Nice point, I forgot to update it (^^;)